sopel/modules/calc.py

```diff
@@ -39,5 +53,8 @@ def c(bot, trigger):
     except SyntaxError:
         bot.reply('Invalid syntax')
         return
+    except ValueError as err:
+        bot.reply("Couldn't run the calculation: %r" % err)
```

Comment:
Nit: I think some readers might stumble on the colon here, since it can be misread as `the calculation, [which was]:`. Potential reword:

```suggestion
        bot.reply("Couldn't run the calculation, got error: %r" % err)
```

Reply:
How about:

```suggestion
        bot.reply("Error running calculation: %r" % err)
```